src/async_command_exec.cpp

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Commit message: 

> - Introduced another class `SpawnActions` which initializes
   `posix_spawn_file_actions_t` inorder safely setup and destroy file
   descriptors.

Typo in class name or missed to update while the changing the class name.